scripts/linters/pylint_extensions.py

```diff
@@ -1674,15 +1679,15 @@ class SingleLineCommentChecker(checkers.BaseChecker):
             'no-capital-letter-at-beginning',
             'Please use capital letter to begin the content of comment.'
         ),
-        'C0019': (
-            'Please use a single space at the beginning of pylint pragmas.',
+        'C0039': (
+            'Please use a single space before pragmas.',
             'no-space-before-pylint-pragma',
-            'Please use a single space at the beginning of pylint pragmas.'
+            'Ensures that there is a space before any pragmas.'
```

Comment:
Suggestion - Ensure single space is present before start of pragma keyword. (A more accurate error/helper msg helps developers to debug code easily)

Reply:
Resolving this because the existing error msg `no-space-at-beginning` should be used via the checker method. This error can be dropped.